ui/src/app/cohort-review/detail-tabs/detail-tabs.component.ts

```diff
@@ -139,156 +156,173 @@ export class DetailTabsComponent implements OnChanges, OnInit, OnDestroy {
     }
   }, {
     name: 'Drugs',
     domain: DomainType.DRUG,
     filterType: PageFilterType.ReviewFilter,
     columns: [
-      itemDate, standardName, standardCode, ageAtEvent, numMentions, firstMention,
-      lastMention, quantity, refills, strength, dataRoute, sourceName, sourceCode,
-      sourceVocabulary, visitId
+      itemDate, standardName, dataRoute, dose, strength, ageAtEvent, numMentions,
+      firstMention, lastMention, visitType
     ],
     reverseEnum: {
       itemDate: itemDate,
       standardName: standardName,
-      standardCode: standardCode,
+      dataRoute: dataRoute,
+      dose: dose,
+      strength: strength,
       age: ageAtEvent,
       numMentions: numMentions,
       firstMention: firstMention,
       lastMention: lastMention,
-      quantity: quantity,
-      refills: refills,
-      strength: strength,
-      route: dataRoute,
-      sourceName: sourceName,
-      sourceCode: sourceCode,
-      sourceVocabulary: sourceVocabulary,
-      visitId: visitId,
+      visitType: visitType,
     }
   }, {
-    name: 'Measurements',
-    domain: DomainType.MEASUREMENT,
+    name: 'Observations',
+    domain: DomainType.OBSERVATION,
     filterType: PageFilterType.ReviewFilter,
     columns: [
-      itemDate, standardName, standardCode, standardVocabulary, valueConcept, valueNumber,
-      valueSource, units, ageAtEvent, labRefRange, sourceName, sourceCode, visitId
+      itemDate, standardName, standardCode, standardVocabulary, ageAtEvent, sourceName,
+      sourceCode, sourceVocabulary, visitId
     ],
     reverseEnum: {
       itemDate: itemDate,
       standardName: standardName,
       standardCode: standardCode,
       standardVocabulary: standardVocabulary,
-      valueConcept: valueConcept,
-      valueNumber: valueNumber,
-      valueSource: valueSource,
-      units: units,
       age: ageAtEvent,
-      labRefRange: labRefRange,
       sourceName: sourceName,
       sourceCode: sourceCode,
+      sourceVocabulary: sourceVocabulary,
       visitId: visitId,
     }
   }, {
-    name: 'Observations',
-    domain: DomainType.OBSERVATION,
+    name: 'Physical Measurements',
+    domain: DomainType.PHYSICALMEASURE,
     filterType: PageFilterType.ReviewFilter,
     columns: [
-      itemDate, standardName, standardCode, standardVocabulary, ageAtEvent, sourceName,
-      sourceCode, sourceVocabulary, visitId
+      itemDate, standardCode, standardVocabulary, standardName, value, unit, ageAtEvent
     ],
     reverseEnum: {
       itemDate: itemDate,
-      standardName: standardName,
       standardCode: standardCode,
       standardVocabulary: standardVocabulary,
+      standardName: standardName,
+      value: value,
+      unit: unit,
       age: ageAtEvent,
-      sourceName: sourceName,
-      sourceCode: sourceCode,
-      sourceVocabulary: sourceVocabulary,
-      visitId: visitId,
     }
   }, {
-    name: 'Physical Measurements',
-    domain: DomainType.PHYSICALMEASURE,
+    name: 'Lab',
```

Comment:
Should this be 'Labs' to match the other tabs?

Reply:
Done.